From 9df5db1621e6375c0d1659c59a2a2d099a470d9e Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Fri, 15 Feb 2019 15:58:30 +0100
Subject: [PATCH 1037/1051] arm64: dts: armada-3900: convert eth port from
 10gbase-kr to 5gbase-kr

Change-Id: I5a49f4033f5af31278f262fcf2374f2ac307af58
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/4402
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Marcin Wojtas <Marcin.Wojtas@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts | 4 ++--
 arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts b/arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts
index 767456ebf97b..ffd334444846 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db-vd-A.dts
@@ -135,7 +135,7 @@
 	status = "okay";
 	/* Network PHY */
 	phy = <&phy0>;
-	phy-mode = "10gbase-kr";
+	phy-mode = "5gbase-kr";
 	/* Generic PHY, providing serdes lanes */
 	phys = <&cp0_comphy2 0>;
 };
@@ -144,7 +144,7 @@
 	status = "okay";
 	/* Network PHY */
 	phy = <&phy1>;
-	phy-mode = "10gbase-kr";
+	phy-mode = "5gbase-kr";
 	/* Generic PHY, providing serdes lanes */
 	phys = <&cp0_comphy4 1>;
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts b/arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts
index 0d199206c22e..f3db91fb51b4 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db-vd-B.dts
@@ -135,7 +135,7 @@
 	status = "okay";
 	/* Network PHY */
 	phy = <&phy0>;
-	phy-mode = "10gbase-kr";
+	phy-mode = "5gbase-kr";
 	/* Generic PHY, providing serdes lanes */
 	phys = <&cp0_comphy2 0>;
 };
-- 
2.17.1

